.circleci/config.yml

```diff
@@ -34,6 +34,8 @@ jobs:
       - run: cargo update
       - run: cargo fetch
       - run: rustc +$(cat rust-toolchain) --version
+      - run: apt-get update
+      - run: apt-get install -y hwloc libhwloc-dev
```

Comment:
can likely be removed now, `libhwloc-dev` is no in the main image

Reply:
Yes, those lines should be removed again.

Reply:
I removed those as I touched the config to make macOS work anyway.